libcipher/caesar.py

```diff
@@ -0,0 +1,107 @@
+__author__ = 'kskrishnasangeeth'
+
+
+class Error(Exception):
+    pass
+
+
+class WrongKey(Error):
+    """ class created for throwing Exception in the event of wrong key being supplied
+
+    Attributes:
+
+        msg -- Explanation on why key is not allowed.
+        key -- Key that is supplied.
+    """
+
+    def __init__(self, msg, key):
+        self.msg = msg
+        self.key = key
+
+
+class NoKeyGiven(Error):
+    """ class created for throwing Exception in the event of no key being supplied
+
+    Attributes:
+
+        msg -- Message to provide key for encryption.
+    """
+
+    def __init__(self, msg):
+        self.msg = msg
+
+
+def cipher_helper(message, key):
+
+    """Function for modularising and removing some of the repeated code.
+
+    Attributes:
+
+        message(str) -- Message that is supplied for encryption/decryption.
```

Comment:
Sorry but you need to follow the convention too the word. A Sphinx plugin called Napoleon will generate documentation from this. See http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html